File: GestionGymAPI/src/main/java/es/neifi/GestionGymAPI/rest/model/registrohorario/RegistroHorario.java
package es.neifi.GestionGymAPI.rest.model.registrohorario;

import java.util.List;

import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.OneToOne;
import javax.persistence.Table;

import com.fasterxml.jackson.annotation.JsonIgnore;

import es.neifi.GestionGymAPI.rest.model.usuario.Usuario;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "registrohorario",schema = "public")
public class RegistroHorario {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id_registrohorario;
	private String horaentrada;
	private String horasalida;
	private String fecha;


}

File: src/app/model/RegistroHorario.ts
import { OnInit } from "@angular/core";

export class RegistroHorario  {

    private id_registrohorario:string;
    private horaentrada:string;
    private horasalida:string
    private fecha:string;
    

    /**
     * Getter $id_registrohorario
     * @return {string}
     */
	public get $id_registrohorario(): string {
		return this.id_registrohorario;
	}

    /**
     * Getter $horaentrada
     * @return {string}
     */
	public get $horaentrada(): string {
		return this.horaentrada;
	}

    /**
     * Getter $fecha
     * @return {string}
     */
	public get $fecha(): string {
		return this.fecha;
	}

    /**
     * Setter $id_registrohorario
     * @param {string} value
     */
	public set $id_registrohorario(value: string) {
		this.id_registrohorario = value;
	}

    /**
     * Setter $horaentrada
     * @param {string} value
     */
	public set $horaentrada(value: string) {
		this.horaentrada = value;
	}

    /**
     * Setter $fecha
     * @param {string} value
     */
	public set $fecha(value: string) {
		this.fecha = value;
	}

    
  }



  

  

  